### PR TITLE
Post: Hardware Accelerated Animations Updates

### PR DESCRIPTION
**Pull request by una**
Adds article about hardware acceleration and updates in Chromium

### CODE REVIEW

**Comment by una** [comment]
This is ready for review now, @kaycebasques. Thank you!

**Comment by una** [comment]
friendly ping!

File: site/en/blog/hardware-accelerated-animations/index.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+---
+title: Updates in Hardware Accelerated Animation Capabilities
```
Comment (kaycebasques) [inline site/en/blog/hardware-accelerated-animations/index.md:2]:
```suggestion
title: Updates in hardware-accelerated animation capabilities
```

File: site/en/blog/hardware-accelerated-animations/index.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+authors:
+  - unakravets
+hero: 'image/HodOHWjMnbNw56hvNASHWSgZyAf2/Tojod6kLLs7q2NCjsi30.jpg'
+alt: >
+  A series of racing bikes
+tags:
+  - chrome-89
+---
+
+*TLDR: Chromium is updating its hardware-acceleration capabilities automatically
```
Comment (kaycebasques) [inline site/en/blog/hardware-accelerated-animations/index.md:16]:
```suggestion
*Summary: Chromium is updating its hardware-acceleration capabilities automatically
```
https://developers.google.com/style/word-list#tldr

File: site/en/blog/hardware-accelerated-animations/index.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+and how it affects the overall performance of your web property. You avoid
+rendering and painting time completely for the animation, meaning smoother
+animations and more performant applications. While Facebook won't be shipping
+this SVG-based loader until browser support for hardware-accelerated SVG is
+greater, it would allow for more flexibility in terms of theming, scaling and
+resolution requirements, and easier maintenance.
+
+## Percentage animations
+
+The Interactions team is also shipping on support for percentage transform animations, shipping in
```
Comment (kaycebasques) [inline site/en/blog/hardware-accelerated-animations/index.md:95]:
```suggestion
The Interactions team is also shipping support for percentage transform animations in
```

File: site/en/blog/hardware-accelerated-animations/index.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+As of Chromium 89, Chrome will join the likes of Firefox to enable hardware-acceleration by
+default on SVG animations. What do you, the developer, need to do? Nothing—this will be
+automatically applied for SVG animations in Chromium 89+.
+
+### Example
+
+Let's take a look at the differences between an SVG animation with and without hardware acceleration turned on. Loading indicators are commonly-used UI elements, such as this one seen on facebook.com. These indicators hint at work being done on the server, while the user waits for a response. In the case shown here, the response would be to load additional results in the sidebar.
+
+<figure>
+  {% img src="image/HodOHWjMnbNw56hvNASHWSgZyAf2/CBSrDDfkO8Ab693jU50K.gif", alt="Facebook sidebar UI", width="305", height="384" %}
```
Comment (kaycebasques) [inline site/en/blog/hardware-accelerated-animations/index.md:64]: Can you convert this to GIF? I know it's extra work but it's important to practice what we preach https://web.dev/replace-gifs-with-videos/

**Review by kaycebasques — APPROVED** [review]

**Comment by una** [comment]
Got this updated and ready to go for Monday

**Comment by robdodson** [comment]
Just noticed this is failing to build because it's using the old shortcode `{% img %}` instead of `{% Img %}`. I can send in a quick PR

**Comment by una** [comment]
Ah, thanks Rob -- I wasn't getting a local error

**Comment by robdodson** [comment]
yeah it's something we changed recently. I don't think we currently have GitHub Checks that will catch a failing build 😅  but Sam and I are discussing it today.

**Pull request merged**